tests/lib/rules/no-warning-comments.js

```diff
@@ -24,15 +24,10 @@ ruleTester.run("no-warning-comments", rule, {
         { code: "// any comment", options: [{ terms: ["fixme", "todo"] }] },
         "// any comment",
         { code: "// any comment", options: [{ location: "anywhere" }] },
-        { code: "// any comment with TODO, FIXME or XXX", options: [{ location: "start" }] },
-        "// any comment with TODO, FIXME or XXX",
```

Comment:
This should be valid as well: `{ location: "start" }` appears to be the rule default. This should only be flagged if the `{ location: "anywhere" }` option is explicitly specified.